### PR TITLE
popup style, style fixes, close filter by click on map or escape button

### PR DESCRIPTION
**Pull request by RostyslavSynenko**

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas,
+    'Courier New', monospace;
+}
+
+.mapboxgl-popup-anchor-top-left .mapboxgl-popup-tip,
+.mapboxgl-popup-anchor-top .mapboxgl-popup-tip,
+.mapboxgl-popup-anchor-top-right .mapboxgl-popup-tip {
+  border-bottom-color: #282c34 !important;
```
Comment (OrestSat) [inline client/src/index.css]: Don't use !important

File: client/src/modules/MapHolder/components/PopupContent/DefibrillatorPopupContent.js [hunk 2]
```diff
@@ -8,17 +8,63 @@ import cancelToken from '../../../../shared/cancel-token';
...
+  title: {
+    color: '#bbb',
+    fontWeight: 'bold'
   }
 });
 
 const DefibrillatorPopupContent = ({ id }) => {
   const classes = useStyle();
   const [currDef, setCurrDef] = useState(null);
+  const titles = {
```
Comment (OrestSat) [inline client/src/modules/MapHolder/components/PopupContent/DefibrillatorPopupContent.js]: Move to consts

File: client/src/modules/Sidebar/components/Header/components/Filter/FormFormik.js [hunk 3]
```diff
@@ -32,6 +32,13 @@ const FormFormik = ({ onSubmit }) => {
     'україномовний',
     'російськомовний'
   ];
+  const informationalPlates = [
```
Comment (OrestSat) [inline client/src/modules/Sidebar/components/Header/components/Filter/FormFormik.js]: Move to consts

File: client/src/index.css (hunk 1 above)
Comment (RostyslavSynenko) [reply client/src/index.css]: All suggestions which I found were to overwrite it with !important. It's the only way to change a tip style. I don't have a direct access to that element and its styles.

**Pull request merged**